.github/workflows/spec_test_on_nuttx.yml

```diff
@@ -127,7 +127,8 @@ jobs:
         wamr_feature_option:
           # Empty option for default
           - { option: "", mode: "" }
-          - { option: "CONFIG_INTERPRETERS_WAMR_GC CONFIG_INTERPRETERS_WAMR_AOT_STACK_FRAME", mode: "-G" }
+          # need to install menhir
+          # - { option: "CONFIG_INTERPRETERS_WAMR_GC CONFIG_INTERPRETERS_WAMR_AOT_STACK_FRAME", mode: "-G" }
```

Comment:
@no1wudi The most recent GC specification tests call for extra dependencies. I attempted to install them but failed, so I've implemented a temporary solution to skip running the GC spec tests on NuttX for the time being. Could you assist in resolving this issue with a new pull request?

Reply:
@no1wudi Kindly remind